backend/baseclass.pm

```diff
@@ -342,10 +343,18 @@ sub _invoke_video_encoder ($self, $pipe_name, $display_name, @cmd) {
     $pipe->blocking(!!$bmwqemu::vars{VIDEO_ENCODER_BLOCKING_PIPE});
 }
 
+sub _ffmpeg_banner () { `ffmpeg 2>&1` // '' }
+
+sub _auto_detect_external_video_encoder ($self) {
+    my $ffmpeg_banner = _ffmpeg_banner;
+    return DEFAULT_FFMPEG_CMD . ' -c:v libsvtav1 -crf 50 -preset 7' if $ffmpeg_banner =~ qr/--enable-libsvtav1(\s|$)/;
+    return DEFAULT_FFMPEG_CMD . ' -c:v libvpx-vp9 -crf 35 -b:v 1500k -cpu-used 1' if $ffmpeg_banner =~ qr/--enable-libvpx(\s|$)/;
```

Comment:
```suggestion
    return join(' ', DEFAULT_FFMPEG_CMD, $bmwqemu::vars{FFMPEG_ARGS} // '-c:v libsvtav1 -crf 50 -preset 7') if $ffmpeg_banner =~ qr/--enable-libsvtav1(\s|$)/;
    return join(' ', DEFAULT_FFMPEG_CMD, $bmwqemu::vars{FFMPEG_ARGS} // '-c:v libvpx-vp9 -crf 35 -b:v 1500k -cpu-used 1') if $ffmpeg_banner =~ qr/--enable-libvpx(\s|$)/;
```

Reply:
I would also not allow this. It makes no sense that both cases would shrink into one case when this variable is specified. If one wanted to customize this then `EXTERNAL_VIDEO_ENCODER_CMD` is the way to go. We could allow `EXTERNAL_VIDEO_ENCODER_FFMPEG_ARGS` in addition so one can just adjust the last part of the command line individually but I would not do this in these special cases. These special cases are really for automatic detection in case one has *not* specified the arguments explicitly and I wouldn't mix the code for adding a customization point into it.

Reply:
ok, leave as is